stumpy/scrump.py

```diff
@@ -46,12 +49,19 @@ def _get_max_order_idx(m, n, orders, start, percentage):
     max_n_dist = 0
     for order_idx in range(orders.shape[0]):
         k = orders[order_idx]
-        max_n_dist = max_n_dist + (n - m + 1 - k)
+        if k >= 0:
```

Comment:
Does this mean that we'll have values of `k` that are less than zero? Are these referring to the lower triangle?

Reply:
Yes that's right